src/app/ecosystem-explorer/page.tsx

```diff
@@ -204,10 +206,7 @@ export default function EcosystemExplorer({ searchParams }: Props) {
       <CTASection
         title="Become Part of the Expanding Ecosystem"
         description="If you’re building on Filecoin and don’t see your project or want to edit your listing, share your details."
```

Comment:
We should ask @jordanlynnf to look at this copy or adapt it ourselves. It reads weird.